options/options.go

```diff
@@ -85,6 +87,8 @@ const (
 	FormatDefault = "default"
 	// FormatRaw specifies that results should be output in raw format.
 	FormatRaw = "raw"
+	// FormatMarkdown specifies that results should be output in Markdown format.
+	FormatMarkdown = "markdown"
```

Comment:
Any discussion where we decided to add `markdown` format as a supported format? Also, maybe keep the support for adding `markdown` separate from this PR?

Reply:
I'm also in favor of this being treated separately, maybe in the Action itself.

Reply:
sync from chat: will remove the markdown support for now, and may add it to the Action in the future